### DISCUSSION TITLE
Write content in the document.

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2171] [Q&A] [answered]
Hi everyone,

I would like after my Brython script has been loaded and executed, to load a .JS file :/
(Yeah during this transition time we still need them ..)

So I was thinking to use something like

```python
document['html'] <= "blah"
```
But of course this is not working :'(

Any ideas ?

Thanks.

**Comment by PierreQuentel** [comment]
```python
from browser import document, html, window

# 1. create a JS script from source code

js = """
window.new_func = function(){
  alert('hello !')
}
"""
document <=  html.SCRIPT(js)

window.new_func()

# 2. load an existing JS script by its url

script = html.SCRIPT(src='jsobj_tests.js')

def load(ev):
    print(window.Rectangle)

script.bind('load', load)
document <= script
```

**Reply by BobSquarePants** [reply]
Thank you @PierreQuentel :)

For my curiosity  I've done also a
```python
document <= '<b>Brython rock !</b>'
```
it work too, but the tag `<b>` are not parsed. meaning I see in the browser

> `<b>Brython rock !</b>`

instead of
> **Brython rock !**

So what does the html module in order to have those input parsed by the browser ?

and is there a way to have an "input" `<=`  parsed when called directly from the Brython script ?

Thanks :)

**Comment by PierreQuentel** [comment]
```python
from browser import document, html

document <= html.B('bold text')
```

**Reply by BobSquarePants** [reply]
Thx :), I knew that.

But I wanted to know if it's possible to create an element (or anything) and being parsed by the browser, without using the html module ?

**Comment by PierreQuentel** [comment, marked as answer]
```python
document.html += '<b>bold text</b>'
document.html += '<br><i>italic</i>'
```

**Comment by BobSquarePants** [comment]
Hi,

How we do the same but for the `<head>` ?

I've tried
```python
document.head += '<link rel="stylesheet" type="text/css" href="some.css">'
```
but this is not working.. any ideas ?

**Reply by BobSquarePants** [reply]
FYI this is working

```python
document.select('head')[0].insertAdjacentHTML('see mdn docs') #https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML
```

but if you have any otherway I'm all ears :)
